### PR TITLE
Chore: Convert "About" page to TailwindCSS

### PR DESCRIPTION
**Pull request by leosoaivan**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behaviour
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc
  - `Fix` - bug fixes

Complete the following checkbox ONLY IF it is applicable to your PR. You can complete it later if it is not currently applicable:
-   [x] I have verified all tests and linters pass against my changes, and/or I have included automated tests where applicable

<hr>

**1. Because:**

Closes https://github.com/TheOdinProject/top-meta/issues/180

**2. This PR:**

- Create a new AboutBelief ViewComponent
- Swaps in TailwindCSS classes as appropriate

### CODE REVIEW

File: app/views/static_pages/about/_overview.html.erb [hunk 1]
```diff
@@ -1,51 +1,54 @@
-<section class="overview odin-dark-bg">
+<section class="py-12 sm:py-24 bg-gray-100">
```
Comment (KevinMulhern) [inline app/views/static_pages/about/_overview.html.erb:1]: [BUG] We'll need `odin-dark-bg` on this to override the gray on dark mode.

```suggestion
<section class="py-12 sm:py-24 bg-gray-100 odin-dark-bg">
```

Screenshot 2022-07-18 at 01 02 14

File: app/views/static_pages/about/_overview.html.erb [hunk 2]
```diff
@@ -1,51 +1,54 @@
...
-    <h2 class="overview__heading accent text-2xl">Overview of The Odin Project</h2>
+    <h2 class="mb-12 text-gold text-center text-2xl font-bold">Overview of The Odin Project</h2>
 
-    <div class="row">
-
-      <div class="col-12 col-md-4 col-lg-4">
-        <div class="odin-stat card-main odin-dark-bg-accent">
-          <p class="odin-stat__content"><%= User.count %></p>
+    <div class="grid grid-cols-1 sm:grid-cols-3 gap-x-8 gap-y-16">
+      <%= render CardComponent.new do |card| %>
```
Comment (KevinMulhern) [inline app/views/static_pages/about/_overview.html.erb:6]: [BUG] The stats cards aren't going full width at below the sm breakpoint:
Screenshot 2022-07-18 at 00 59 14

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

**Comment by KevinMulhern** [comment]
Looking great @leosoaivan 💪 

Just some minor stuff I picked up on.

File: app/components/about_belief_component.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<div class="flex-row sm:grid sm:grid-cols-12 pb-14">
+  <div class="flex sm:col-span-2 sm:pr-4 justify-center ">
+    <%= image_tag @belief[:image_path], alt: @belief[:image_alt] %>
+  </div>
+
+  <div class="sm:col-span-10 sm:pl-4 text-center sm:text-left">
```
[the remark this answers is not in the document]
Comment (leosoaivan) [reply app/components/about_belief_component.html.erb:6]: _smaaaart_ 👍 😆

File: app/components/about_belief_component.html.erb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <h3 class="pt-10 sm:pt-0 pb-4 text-xl">
```
[the remark this answers is not in the document]
Comment (leosoaivan) [reply app/components/about_belief_component.html.erb]: TIL 👏

File: app/views/static_pages/about.html.erb [hunk 5]
```diff
@@ -40,7 +40,7 @@
       <div class="beliefs">
         <div class="col-lg-8 offset-lg-2">
           <h2 class="bold text-center beliefs-heading text-2xl">Our Beliefs</h2>
```
[the remark this answers is not in the document]
Comment (leosoaivan) [reply app/views/static_pages/about.html.erb]: Had it at mb-16, but I think mb-10 looks better. Changing.

File: app/views/static_pages/about/_overview.html.erb [hunk 6]
```diff
@@ -1,51 +1,54 @@
...
+        <% card.body(classes: 'py-4 text-center' ) do %>
```
[the remark this answers is not in the document]
Comment (leosoaivan) [reply app/views/static_pages/about/_overview.html.erb:7]: On one hand, totally agree. On the other, it creates an issue with the "Open source" text and image:

image

How do you want to proceed?

[the remark this answers is not in the document]
Comment (leosoaivan) [reply app/views/static_pages/about/_overview.html.erb:7]: image

[the remark this answers is not in the document]
Comment (KevinMulhern) [reply app/views/static_pages/about/_overview.html.erb:7]: Smaller font looks great to me 👍

**Review by KevinMulhern — APPROVED** [review]
Nice work @leosoaivan this looks/feels solid 💪

File: app/views/static_pages/about.html.erb [hunk 7]
```diff
@@ -40,7 +40,7 @@
       <div class="beliefs">
         <div class="col-lg-8 offset-lg-2">
```
[the remark this answers is not in the document]
Comment (leosoaivan) [reply app/views/static_pages/about.html.erb:41]: Ah gotcha. Missed those.

**Review by KevinMulhern — APPROVED** [review]
Thanks for all your work on this @leosoaivan  🚀

**Pull request merged**